### PR TITLE
adding source folder

### PR DESCRIPTION
**Pull request by narinder17**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-13104

### Change description ###
adding sources folder for golden path update 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -399,7 +399,7 @@ To roll back, do the following
 
   <details>
 
-  <summary>Azure Firewall disassociate PIP</summary>
+  <summary>AzFw disassociate pip</summary>
```
Comment (madjava) [inline README.md:402]: leave this as `Azure Firewall` lets not assume everyone knows what `AzFw` means 😜

File: README.md [hunk 2]
```diff
@@ -384,7 +384,7 @@ entries as described above.
 ### Points to note when going through the AKS steps
 
 - Make sure using <labs-YourGitHubUsername-nodejs> when configuring both Public DNS/Private DNS for Application
-- There maybe needed some troubleshooting e.g. library versions may be deprecated or need updating. 
+- There maybe needed some troubleshooting e,g version deprecated/update.
```
Comment (madjava) [inline README.md:387]: update e,g to e.g

**Comment by madjava** [comment]
@narinder17 Why duplicate the images folder?

**Review by madjava — APPROVED** [review]

**Comment by madjava** [comment]
Some updates/changes required but lets get the base changes in. Please merge 💪🏽

File: README.md (hunk 2 above)
Comment (narinder17) [reply README.md:387]: updated

File: README.md (hunk 1 above)
Comment (narinder17) [reply README.md:402]: Changed abbrievation to full name

**Pull request merged**